e2e-tests/expose-sharded/run

```diff
@@ -0,0 +1,232 @@
+		"userAdmin:userAdmin123456@$cluster-mongos.$namespace"
+	run_mongo 'db.createUser({user: "myApp", pwd: "myPass", roles: [{ db: "myApp", role: "readWrite" }]})' \
+		"userAdmin:userAdmin123456@$cluster-rs0-0.$cluster-rs0.${namespace}" "mongodb"
+
+	run_mongos \
+		'sh.enableSharding("myApp","rs0")' \
+		"clusterAdmin:clusterAdmin123456@$cluster-mongos.$namespace"
+	run_mongos \
+		'use myApp\n db.test.insert({ x: 100500 })' \
+		"myApp:myPass@$cluster-mongos.$namespace"
+	sleep 10 # for minikube
+	compare_mongos_cmd "find" "myApp:myPass@$cluster-mongos.$namespace"
+
+	desc 'Unexposed -> Exposed, ClusterIP'
+	expose_cluster "ClusterIP"
+
+	wait_for_running $cluster-rs0 3
+	wait_for_running $cluster-cfg 3 "false"
+	wait_for_running $cluster-mongos 3
+	run_mongos \
+		'use myApp\n db.test.insert({ x: 100501 })' \
+		"myApp:myPass@$cluster-mongos.$namespace"
+
+	sleep 10 # for minikube
+
+	compare_mongos_cmd "find" "myApp:myPass@$cluster-mongos.$namespace" "-2nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-0.$cluster-rs0.$namespace" "-2nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-1.$cluster-rs0.$namespace" "-2nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-2.$cluster-rs0.$namespace" "-2nd"
+	compare_mongo_config ${cluster} $namespace
+
+	desc 'Exposed, ClusterIP -> LoadBalancer'
+	expose_cluster "LoadBalancer"
+
+	wait_for_running $cluster-rs0 3
+	wait_for_running $cluster-cfg 3 "false"
+	wait_for_running $cluster-mongos 3
+	run_mongos \
+		'use myApp\n db.test.insert({ x: 100502 })' \
+		"myApp:myPass@$cluster-mongos.$namespace"
+	sleep 10 # for minikube
+
+	compare_mongos_cmd "find" "myApp:myPass@$cluster-mongos.$namespace" "-3nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-0.$cluster-rs0.$namespace" "-3nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-1.$cluster-rs0.$namespace" "-3nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-2.$cluster-rs0.$namespace" "-3nd"
+	sleep 60
+	compare_mongo_config ${cluster} $namespace
+
+	desc "Pause Exposed cluster (LoadBalancer)"
+	stop_cluster ${cluster}
+	start_cluster ${cluster}
+
+	run_mongos \
+		'use myApp\n db.test.insert({ x: 100504 })' \
+		"myApp:myPass@$cluster-mongos.$namespace"
+	sleep 10 # for minikube
+
+	compare_mongos_cmd "find" "myApp:myPass@$cluster-mongos.$namespace" "-5nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-0.$cluster-rs0.$namespace" "-5nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-1.$cluster-rs0.$namespace" "-5nd"
+	compare_mongo_cmd "find" "myApp:myPass@$cluster-rs0-2.$cluster-rs0.$namespace" "-5nd"
+
+	compare_mongo_config ${cluster}
+
+	desc "Exposed,  LoadBalancer -> ClusterIP"
+	expose_cluster "ClusterIP"
+
+	wait_for_running $cluster-rs0 3
+	wait_for_running $cluster-cfg 3 "false"
+	wait_for_running $cluster-mongos 3
+	run_mongos \
+		'use myApp\n db.test.insert({ x: 100503 })' \
+		"myApp:myPass@$cluster-mongos.$namespace"
+	sleep 10 # for minikube
```

Comment:
I propose to check if we run on minikube and only in that case to do sleep. Maybe like this: https://github.com/percona/percona-server-mysql-operator/blob/main/e2e-tests/vars.sh#L33-L35